semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DocumentOps.scala

```diff
@@ -16,14 +16,14 @@ trait DocumentOps { self: DatabaseOps =>
     if (!g.settings.Yrangepos.value) {
       sys.error("the compiler instance must have -Yrangepos enabled")
     }
-    if (g.useOffsetPositions) {
-      sys.error("the compiler instance must use range positions")
-    }
+    // for some reason this fails with ammonite, even though the first Yrangepos == true
+//    if (g.useOffsetPositions) {
+//      sys.error("the compiler instance must use range positions")
+//    }
     if (!g.settings.plugin.value.exists(_.contains("semanticdb"))) {
       sys.error("the compiler instance must use the semanticdb plugin")
     }
     if (!g.analyzer.getClass.getName.contains("HijackAnalyzer")) {
-      println(g.analyzer.getClass.getName)
```

Comment:
Can we maybe include the class name in the error message? It's really helpful for debugging what might be wrong.

Reply:
Addressed abe5531